content/zh/docs/devops-user-guide/how-to-use/jenkins-shared-library.md

```diff
@@ -1,74 +1,75 @@
+1. [登录 Jenkins 仪表板](https://kubesphere.io/zh/docs/devops-user-guide/how-to-use/jenkins-setting/#log-in-to-jenkins-to-reload-configurations)并点击左侧导航栏中的**系统管理**
 
-2.  Scroll down and click **Configure System**.
+2. 向下滚动并点击**系统配置**。
 
-   ![click_configure](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/click-configure.png)
+   ![click_configure](/images/docs/zh-cn/devops-user-guide/use-devops/jenkins-shared-library/click-configure.png)
 
-3. Scroll down to **Global Pipeline Libraries** and click **Add**.
+3. 向下滚动到 **Global Pipeline Libraries**，然后点击**新增**。
 
-   ![click-add](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/click-add.png)
+   ![click-add](/images/docs/zh-cn/devops-user-guide/use-devops/jenkins-shared-library/click-add.png)
 
-4. Configure the fields as below.
+4. 配置字段如下所示。
 
-   - **Name**. Set a name (for example, `demo-shared-library`) for the shared library so that you can import the shared library by referring to this name in a Jenkinsfile.
+   - **Name：** 从为共享库设置名称（例如，``demo-shared-library``），以便可以通过在 Jenkinsfile 中引用此名称来导入共享库。
 
-   - **Default version**. Set a branch name from the repository where you put your shared library as the default branch for importing your shared library. Enter `master` for this tutorial.
+   - **Default version：** 从将共享库放在其中的仓库中设置分支名称，作为导入共享库的默认仓库分支。本教程将使用 master。
 
-   - Under **Retrieval method**, choose **Modern SCM**.
+   - 在 **Retrieval method** 下，选择 **Modern SCM**。
 
-   - Under **Source Code Management**, choose **Git** and enter the URL of the example repository for **Project Repository**. You have to configure **Credentials** if you use your own repository that requires the credentials for accessing it.
+   - 在 **Source Code Management** 下，选择 **Git** 并为项目仓库输入**示例仓库**的 URL 。如果您使用自己的仓库则需要配置**访问凭据**。
 
-     ![configure-shared-library](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/configure-shared-library.png)
+     ![configure-shared-library](/images/docs/zh-cn/devops-user-guide/use-devops/jenkins-shared-library/configure-shared-library.png)
 
-5. When you finish editing, click **Apply**.
+5. 当您结束编辑，点击**应用**
 
    {{< notice note >}}
 
-   You can also configure [Folder-level Shared Libraries](https://www.jenkins.io/doc/book/pipeline/shared-libraries/#folder-level-shared-libraries).
+   您还可以配置[文件夹级别的共享库](https://www.jenkins.io/zh/doc/book/pipeline/shared-libraries/#folder-level-shared-libraries)。
 
    {{</ notice >}}
 
-## Use the Shared Library in a Pipeline
+## 在流水线中使用共享库
 
-### Step 1: Create a pipeline
 
-1. Log in to the KubeSphere web console as `project-regular`. Go to your DevOps project and click **Create** on the **Pipelines** page.
+### 步骤 1： 创建流水线
 
-2. Set a name (for example, `demo-shared-library`) in the pop-up window and click **Next**.
+1. 用 ``project-regular`` 帐户登录 KubeSphere web 控制台。进入 DevOps 工程并点击**流水线**页面上的**创建**。
```

Comment:
I think there are extra backticks(`) before and after the word project-regular. Please check and delete them.